### DISCUSSION TITLE
Infinity Ham - please add to Feeding Upgrade

### DISCUSSION THREAD

**Discussion by seals55** [P3pp3rF1y/SophisticatedBackpacks#1115] [Ideas]
Similar to artifacts:eternal_steak, please add the [Relics - Infinity Ham](https://www.curseforge.com/minecraft/mc-mods/relics-mod) to valid food for the (advanced) Feeding Upgrade
relics:infinity_ham

**Comment by P3pp3rF1y** [comment]
Sounds like it's not marked as food and potentially doesn't have a usual way of eating so it would likely require very special integration which I don't have for any other food yet. Not really in favor of doing that, would make much more sense for that mod to just have this be a normal with its own properties and then it would be automatically supported

**Comment by P3pp3rF1y** [comment]
And I actually took an extremely quick look at the item's source code and there's a lot that's happening there so it's both unlikely this will ever be supported in feeding upgrade as it is unlikely for Relics to make into more regular food like the eternal steak.

**Reply by seals55** [reply]
Understood, thanks for looking into it!
